internal/gitstream/owners.go

```diff
@@ -0,0 +1,27 @@
+package gitstream
+
+import (
+	"context"
+	"math/rand"
+
+	"github.com/google/go-github/v47/github"
+)
+
+type Owners struct {
+	Approvers []string `yaml:"approvers"`
+	Reviewers []string `yaml:"reviewers"`
+	Component string   `yaml:"component"`
+}
+
+func (o *Owners) getAssignee(ctx context.Context, gc *github.Client, user *github.User) (string, error) {
```

Comment:
I think `user` here should also be of type `string`.

Reply:
Fixed.